### PR TITLE
Feat fix this #397

### PR DESCRIPTION
**Pull request by AccaliaDeElementia**

### CODE REVIEW

File: lib/commands.js [hunk 1]
```diff
@@ -547,10 +548,13 @@ exports.bindCommands = function bindCommands(forum) {
                 if (!checkAvailable(forbiddenCmds, cmd, 'error', forbiddenMsg)) {
                     return reject(new Error('E_FORBIDDEN_COMMAND'));
                 }
+                if (context === undefined) {
+                    context = new Object();
```
Comment (yamikuronue) [inline lib/commands.js]: Is that a breaking change? I'm not sure what the old value of `this` would have been.

Comment (AccaliaDeElementia) [inline lib/commands.js]: yep. from the checkin comment (and thus the changelog): 

Any command that relies on the previously undocumented value of `this` within a command handler will be broken by this update.

Comment (yamikuronue) [inline lib/commands.js]: ah okay, cool. That'll trigger correctly for semrel?

Comment (AccaliaDeElementia) [inline lib/commands.js]: on its own, no. I got the syntax wrong in the initial commit, but i corrected that in the comments for 617406a

i could have rewritten the history and pushed a new branch.... but i chickened out on that becuase rewriting history is scary.

Comment (yamikuronue) [inline lib/commands.js]: That's fair. That explains why I didn't see it before :) 

👍 Looks good!

**Pull request merged**